src/test/Microsoft.Extensions.DependencyModel.Tests/JsonAssetions.cs

```diff
@@ -31,7 +31,7 @@ public JsonAssetions(JToken token)
             var token = Subject[expected];
             Execute.Assertion
                 .ForCondition(token != null)
-                .FailWith($"Expected {Subject} to have property '" + expected + "'");
+                .FailWith("Expected {0} to have property '{1}'", Subject, expected);
```

Comment:
Why this change?  How about converting it all to interpolated string usage?

Reply:
Previously, if a test constraint fails, it resulted in an exception due to the way fluentassertions handles args, so it would just show the exception instead of the test assertion info:
```
System.FormatException : Input string was not in a correct format.

System.Text.StringBuilder.FormatError()
System.Text.StringBuilder.AppendFormatHelper(IFormatProvider provider, String format, ParamsArray args)
System.String.FormatHelper(IFormatProvider provider, String format, ParamsArray args)
System.String.Format(String format, Object[] args)
FluentAssertions.Execution.MessageBuilder.FormatArgumentPlaceholders(String failureMessage, Object[] failureArgs) in C:\projects\fluentassertions-vf06b\Src\Core\Execution\MessageBuilder.cs: line 66
FluentAssertions.Execution.MessageBuilder.Build(String message, Object[] messageArgs, String reason, ContextDataItems contextData) in C:\projects\fluentassertions-vf06b\Src\Core\Execution\MessageBuilder.cs: line 42
FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args) in C:\projects\fluentassertions-vf06b\Src\Core\Execution\AssertionScope.cs: line 190
Microsoft.Extensions.DependencyModel.Tests.JsonAssetions.HaveProperty(String expected) in D:\git\coresetup15\src\test\Microsoft.Extensions.DependencyModel.Tests\JsonAssetions.cs: line 32
```